Profiles - ZIB 2017/zib-Wound-MedicalDeviceProduct.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Best practice to add a "purpose" field describing how this profile relates to the HCIM

Reply:
done